onnx/defs/operator_sets.h

```diff
@@ -979,4 +981,25 @@ inline void RegisterOnnxOperatorSetSchema() {
   RegisterOpSetSchema<OpSet_Onnx_ver14>();
 }
 
+inline void RegisterOnnxOperatorSetSchema(int target_version) {
+  // Sets to record the loaded version/ prevent the full operator check in Debug mode
+  OpSchemaRegistry::Instance()->SetLoadedSchemaVersion(target_version);
+  // Keep ordered by descending; return early if target_version has been reached
```

Comment:
can you mention in the comment that the calls to schema registration are required to be in descending order for this to work correctly